javascript-data-model/order-history.js

```diff
@@ -1,79 +1,75 @@
   {
     placed: 'August 4, 2020',
-    total: '$34.00',
+    total: 34.00,
     to: 'JS Masher',
     order: '114-3941689-8772232',
     details: '',
     invoice: '',
     delivered: 'Aug 8, 2020',
-    item: {
-      image: [''],
-      title: ['Javascript for impatient programmers'],
-      author: ['Rauschmayer, Dr. Axel'],
-      return: ['Return window closed on Sep 7, 2020'],
-      price: ['$31.55']
-    },
-    again: '',
-    view: '',
-    review: ''
+    item: [
+      {
+        image: [''],
+        title: ['Javascript for impatient programmers'],
+        author: ['Rauschmayer, Dr. Axel'],
+        return: ['Return window closed on Sep 7, 2020'],
+        price: [31.55]
+      }
+    ]
   },
   {
     placed: 'July 19, 2020',
-    total: '$44.53',
+    total: 44.53,
     to: 'JS Masher',
     order: '113-9984268-1280257',
     details: '',
     invoice: '',
     delivered: 'Jul 20, 2020',
     status: 'Your package was delivered. It was handed directly to a resident.',
-    item: {
-      image: [''],
-      title: ['The Timeless Way of Building'],
-      author: ['Alexander, Christopher'],
-      return: ['Return window closed on Aug 19, 2020'],
-      price: ['41.33']
-    },
-    again: '',
-    view: '',
-    review: ''
+    item: [
+      {
+        image: [''],
+        title: ['The Timeless Way of Building'],
+        author: ['Alexander, Christopher'],
+        return: ['Return window closed on Aug 19, 2020'],
+        price: [41.33]
+      }
+    ]
   },
   {
     placed: 'July 4, 2020',
-    total: '$17.22',
+    total: 17.22,
     to: 'JS Masher',
     order: '114-2875557-9059409',
     details: '',
     invoice: '',
     delivered: 'Jul 7, 2020',
     status: 'Your package was delivered. It was handed directly to a resident',
-    item: {
-      image: [''],
-      title: ['Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Apater---'],
-      return: ['Return window closed on Aug 5, 2020'],
-      price: ['$15.98']
-    },
-    again: '',
-    view: '',
-    review: ''
+    item: [
+      {
+        image: [''],
+        title: ['Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Apater---'],
+        return: ['Return window closed on Aug 5, 2020'],
+        price: [15.98]
+      }
+    ]
   },
   {
     placed: 'July 5, 2020',
-    total: '$138.93',
+    total: 138.93,
     to: 'JS Masher',
     order: '113-2883177-268248',
     details: '',
     invoice: '',
     delivered: 'Jul 5, 2020',
-    item: {
-      images: ['', ''],
-      title: ['GameCube Controller', 'The Art of Sql'],
-      return: ['Return window closed on Aug 4, 2020'],
-      price: ['$94.95, 33.99']
-    },
-    again: '',
-    view: '',
-    support: '',
-    review: ''
+    item: [
+      {
+        images: ['', ''],
+        title: ['GameCube Controller', 'The Art of Sql'],
+        author: ['Farlout, Stephanie'],
+        return: ['Return window closed on Aug 4, 2020'],
+        price: [94.95, 33.99]
```

Comment:
Instead of one object holding arrays, this should be broken up into two objects in the array, one for each item.